js/screens/welcome-screen.js

```diff
@@ -3,3 +3,51 @@ export const welcomeScreen = () => {
    welcomeStepWrapper.innerText = 'Welcome screen';
    return welcomeStepWrapper;
 };
+
+function orangeButton() {
+   const chooseButton = document.createElement('button');
+   const contentButton = document.innerText('Choose');
+   const srcButton = document.src('./components/choose.svg');
+   document.chooseButton.appendChild(contentButton && srcButton);
+   return chooseButton;
+}
+
+/*
+button.addEventListener('click', function (event) {});
+*/
+
+image.onClick = function Choose() {
+   image.src = './components/choose.svg';
+   return image.src;
+}
+
+function UltimatePixelChampionships(fontWelcomeScreen, fontForm) {
```

Comment:
**nitpick:** function names should be written with `camelCase` if they are not classes. Only classes are written in `PascalCase` in JS